application/libraries/Librivox_search.php

```diff
@@ -103,6 +103,24 @@ function advanced_title_search($params)
 						OR ( st.section_language_id IS NULL AND p.language_id = ' . $recorded_language . ') ) ';
 			}
 		}
+		elseif ($recorded_language === -1)
```

Comment:
And then this could be way more readable, instead of being left wondering why -1 is magic ;)